anchor/src/instruction.rs

```diff
@@ -85,6 +89,19 @@ accounts_struct! {
             is_signer: false,
             is_writable: false,
         },
+        pub from_account {
+            is_signer: false,
```

Comment:
Doesn’t the from account need to sign? Or at least its owner?

Reply:
Yeah, it was misnamed as `reserve_authority`, I replaced it for `user_authority`.